.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,98 @@
+"Question" template. If you need help opening an issue, see
+[this](https://nwchemex-project.github.io/.github/resources/github/issue.html)
+tutorial.
+
+## Ways to Contribute
+
+There's three main ways to contribute:
+
+1. Reporting Bugs
+
+   NWChemEx uses GitHub's issue feature for bug tracking. If you think you have
+   found a bug, please first search the issues to see if it has already been
+   reported. If not, feel free to open a new issue using the "Bug report"
+   template.
+
+2. Suggesting Features
+
+    We're happy to consider implementing new features. Just be aware most of us
+    are not directly paid to do software maintenance on NWChemEx, which means it
+    may take us a while to implement the feature. Like bugs, we use GitHub's
+    issue feature for tracking feature requests. Please do a quick search to
+    make sure someone else hasn't already suggested the feature. If there is an open issue, feel free to
+    add a comment in that issue. This will help us to track the need for a feature. If the feature
+    is indeed new, then please open a new issue using the "Feature request"
+    template.
+
+3. Contributing Code
+
+    Please note that NWChemEx is a large project, with many moving pieces. We
+    are also very concerned with performance. This requires planning, and
+    maintaining a unified design. Until you are familiar with the inner workings
+    of NWChemEx and the overall design, we **strongly** recommend chatting with
+    a more established developer **before** spending any extensive time writing
+    code. The best way to do this is by opening a draft pull request (PR) and
+    articulating what you intend to do, and how you intend to do it. While we
+    strive to be an open and welcoming community, the size of the project
+    mandates that we adhere to the unifying design to keep things on track. PRs
+    will need to be aligned with this design before they can be merged.
+
+    That all said, we understand that this may seem daunting for new developers.
+    We also understand it takes time to come to grips with the project. We
+    consider ourselves a welcoming and helpful community and are more than
+    willing to walk you through the process and answer any questions you may
+    have. Just remember we all started somewhere.
+
+    Also if you're just committing a small change, no need to run it by us. Just open the (non-draft) PR with the change!
+
+    For help on creating a pull request see [this](https://nwchemex-project.github.io/.github/resources/github/pull_request.html) tutorial.
+
+## Contributor License Agreement (CLA)
+
+Since the NWChemEx project is developed primarily by the U.S. Department of
+Energy, you will need sign a CLA for legal reasons. When you make a PR, you will
+automatically be prompted to view and sign the CLA (if you have  not already).
+The contents of the CLA are:
+
+```
+By contributing computer software code to the CMakePP project I certify that the
```

Comment:
Presumably you'll want a generic placeholder here rather than `CmakePP`

Reply:
whoops. Good catch.